src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The excavator has started work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The excavator stops working");

```

Comment:
Similarly to the `doWork` method, the message here should be more informative. Indicate that it is the Excavator that stops working.